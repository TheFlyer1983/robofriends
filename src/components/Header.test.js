import { render } from '@testing-library/react';
import React from 'react';
import Header from './Header';

it('Loads header properly', () => {
  const { container, rerender } = render(<Header />);
  expect(container).toMatchSnapshot();
  rerender(<Header />);
  expect(container).toMatchSnapshot();
});
