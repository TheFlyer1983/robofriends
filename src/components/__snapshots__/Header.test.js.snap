// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loads header properly 1`] = `
<div>
  <div>
    <h1
      class="f1"
    >
      RoboFriends
    </h1>
    <button
      color="red"
    >
      Count: 
      0
    </button>
  </div>
</div>
`;

exports[`Loads header properly 2`] = `
<div>
  <div>
    <h1
      class="f1"
    >
      RoboFriends
    </h1>
    <button
      color="red"
    >
      Count: 
      0
    </button>
  </div>
</div>
`;
